### ISSUE TITLE
[Bug] - Better Tooltips Client Crash

### ISSUE THREAD

**Issue by serainox420** [MeteorDevelopment/meteor-client#2212]
### Describe the bug

Better Tooltips Crash Exploit.
A bug that crashes client when hovered over certain modified items.

### Steps to reproduce

```bash
Generate written book with custom event text and leave text as blank, hovering over it crashes client.
Generate suspicious stew with effect 0 or less / or 33 or more and hover over it with Better Tooltips enabled to crash client.
```

### Meteor Version

0.4.7 1381

### Minecraft Version

1.18.1

### Operating System

Linux

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by serainox420** [comment]
Use this to generate crash soup.
`/give @s suspicious_stew{Effects:[{EffectDuration:700,EffectId:33b}],display:{Name:'{"text":"BT Crash Soup"}'}}`

Use this to generate crash book.
`/give @s written_book{RepairCost:0,author:"",display:{Name:'{"text":"BT Crash Book"}'},pages:['{"clickEvent":{"action":"run_command","value":"/say 666"}'],resolved:1b,title:""}`

**Comment by MineGame159** [comment]
Fixed in dd517b59e85c4b00dea48c7a5955c41ac029fd6f.
